package.json

```diff
@@ -34,7 +34,7 @@
     "@mui/material": "5.10.14",
     "@mui/styles": "5.10.14",
     "@mui/system": "5.10.14",
-    "@reduxjs/toolkit": "^1.9.5",
+    "@reduxjs/toolkit": "^2.0.1",
```

Comment:
@okotsopoulos  - I'm really not sure why this was updated. Maybe we need to remove the carrot?